crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -763,7 +838,20 @@ module deepbook::clob_v2 {
                         // This guarantees that the subtraction will not underflow
                         filled_quote_quantity - filled_quote_quantity_without_commission,
                         maker_rebate
-                    )
+                    );
+                    vector::push_back(
```

Comment:
same question here about potential vector size

Reply:
That's a good point! Since orders are stored as df in clob_v2, as per sui limits only upto a total of 1k dfs could be read in a single PTB. So, we can assume that number of order matches in a single txn would be less 1k(before df limit is hit), hence scalability of vector to store metadata is taken care off. Let me know if you see other concerns about scalability of vector. Thanks!